<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryPage</class>
 <widget class="QWidget" name="InventoryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>858</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory Management</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QGroupBox" name="logEntryPanel">
     <property name="title">
      <string>Log Entry Panel</string>
     </property>
     <layout class="QGridLayout" name="logEntryLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="diameterLabel">
        <property name="text">
         <string>Diameter (in):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="costLabel">
        <property name="text">
         <string>Cost ($):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="lengthIn"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="diameterIn"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="lengthFt"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="speciesEntry"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="costValue"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="speciesLabel">
        <property name="text">
         <string>Species:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="qualityValue">
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPushButton" name="addLogButton">
        <property name="text">
         <string>Add Log</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lengthInLabel">
        <property name="text">
         <string>Inches:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lengthFtLabel">
        <property name="text">
         <string>Length (ft):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="qualityLabel">
        <property name="text">
         <string>Quality (1â€“5):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Drying:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="dryingComboBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="locationEntry"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="inventoryTabs">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="logsTab">
      <attribute name="title">
       <string>Logs</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Species:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="logSpeciesComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Length (Min):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="logLengthMin">
           <property name="suffix">
            <string> ft</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Length (Max):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="logLengthMax">
           <property name="suffix">
            <string> ft</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Radius (Min):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="logRadiusMin">
           <property name="suffix">
            <string> in</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Radius (Max):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="logRadiusMax">
           <property name="suffix">
            <string> in</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Drying:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="logDryingComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="logsTableView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cookiesTab">
      <attribute name="title">
       <string>Cookies</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="cookiesTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="slabsTab">
      <attribute name="title">
       <string>Live Edge Slabs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="slabsTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lumberTab">
      <attribute name="title">
       <string>Lumber</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="lumberTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="firewoodTab">
      <attribute name="title">
       <string>Firewood</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="firewoodTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="detailedViewCheckBox">
     <property name="text">
      <string>Detailed View</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="importLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="spreadsheetImporterButton">
       <property name="text">
        <string>Import from Spreadsheet...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
